kuzzleio/kuzzle
# Allow to delete a still-used profile by removing it from assigned users

## What does this PR do ?
Adds an option to **security:deleteProfile** so it can update associated users (remove the required profile from their _profileIds_ list) when removing a profile.

If a user only had one profile, and it happens to be the one that is getting removed: the _Anonymous_ profile will be assigned.

closes #1683 

### How should this be manually tested?

- Create a user and give it a custom profile
- Remove the profile
- Check that the user got updated
